### PR TITLE
Tweaked package, added build steps to readme

### PR DESCRIPTION
**Pull request by jasonk000**
Building it and installing locally required few extra steps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "async": "^1.5.2",
     "aws-sdk": "^2.3.0",
+    "babel-preset-es2015": "^6.6.0",
```
Comment (SomeoneWeird) [inline package.json:32]: this is already in dev-dependencies

File: README.md [hunk 2]
```diff
@@ -134,3 +134,17 @@ You can prompt for a value when you run `create` or `update`.
...
+
+To build on your local:
+
+```
+$ git clone ...
+$ cd cirrus/
+$ sudo npm install babel-cli
+$ npm install
+$ npm run build
+$ sudo npm install -g .
```
Comment (SomeoneWeird) [inline README.md:147]: you shouldn't need to do this

File: README.md [hunk 3]
```diff
@@ -134,3 +134,17 @@ You can prompt for a value when you run `create` or `update`.
...
 }
 ```
+# Building
+
+To build on your local:
+
+```
+$ git clone ...
+$ cd cirrus/
+$ sudo npm install babel-cli
```
Comment (SomeoneWeird) [inline README.md:144]: you shouldn't need to do this

**Comment by SomeoneWeird** [comment]
If you do like `npm link --local` then it will build + symlink globally too

**Comment by jasonk000** [comment]
not required

**Pull request closed without merging**